### ISSUE TITLE
fails to run obcommand that has spaces

### ISSUE THREAD

**Issue by gabrielcarloto** [esm7/obsidian-vimrc-support#169]
**Please Review Before Posting!**

- [x] I checked that the bug *does not happen* in the [CodeMirror Vim demo](https://codemirror-vim--util.repl.co/). If it does, please report it [there](https://github.com/replit/codemirror-vim) and not here.
- [x] I'm reasonably sure that this bug is indeed about the Vimrc file support and not a general Vim in Obsidian issue. If it's a general Vim issue, report it [here](https://forum.obsidian.md/c/bug-reports/7).

**Describe the bug:**

I want to map a templater template to a key. I added it to the obsidian commands using the Template Hotkeys section in the templater settings. The command id is `templater-obsidian:path/to/template.md`. In my case, the templates folder has a space, therefore the id looks like this: `templater-obsidian:path to/template.md`. I added the following lines to my config:

```vim
exmap insert_template obcommand templater-obsidian:path to/template.md
nmap <Space>t :insert_template
```

When I press `<Space>t`, it only outputs in the console `Error: obcommand requires exactly 1 parameter`. 

Is there a way to make it work (without having to rename the folder)? If not, I'd suggest adding support for `"command:that has spaces"`.

**Environment (please complete the following information):**

 - OS: Windows 11
 - Vimrc plugin version: 0.9.0

**Comment by jiyee** [comment]
I don't think this can be fixed right now because it's a limitation of CodeMirror Vim and you can't customize the delimiters; however, I found a way around it. You can use the `Command Alias` plugin, customize a new command, and find the new command id without space through `:obcommand` (the command id like this obsidian-command-alias-plugin: Alias: XXXXXXXXX) , and map the new command directly.

**Comment by gabrielcarloto** [comment]
Never heard of that plugin. It works well, thank you!
